src/components/structures/MessagePanel.js

```diff
@@ -368,6 +369,15 @@ module.exports = React.createClass({
         this.eventNodes[eventId] = node;
     },
 
+    // once dynamic content in the events load, make the scrollPanel check the
+    // scroll offsets.
+    _onWidgetLoad: function() {
+        var scrollPanel = this.refs.scrollPanel;
+        if (scrollPanel) {
+            scrollPanel.checkScroll();
```

Comment:
okay, have changed the gemini forceUpdate to the whole scrollpanel forceUpdate. this feels a bit vicious, but i assume you know what you're asking for :)